panel/widgets/indicators.py

```diff
@@ -18,6 +18,7 @@
 from __future__ import annotations
 
 import math
+from multiprocessing import Lock
```

Comment:
`multiprocessing` can't be imported in Pyodide environments so we have to find a different solution here or guard the import.